From 51ec7323d405b0890e4c4e5bfd93a476efd02cbf Mon Sep 17 00:00:00 2001
From: sai surya teja <sssteja@habana.ai>
Date: Mon, 01 Aug 2022 19:24:36 +0300
Subject: [PATCH] [SW-97583] gaudi2: remove usage of secured pcie ids

Its decided not to support two different dev ids (secured and
non-secured) for gaudi2. LKD will rely on boot_status sec_en bit to
identify if FW security is enabled.

This patch removes the logic to change the dev id if security is enabled

Change-Id: I4b0631a3a333b8d5f19c71b5883dbfe8f7890704
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 1afcca4..7ad955d 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -329,18 +329,8 @@
 
 void preboot_device_id_init(void)
 {
-	pci_dev_id = PCI_IDS_GAUDI2;
-
-	/*
-	 * Secure pci ids:
-	 * PCI_IDS_GAUDI2_SEC : 0x1030
-	 */
-	/* If security is enabled set the PCI ID accordingly*/
-	if (preboot_security_is_enabled())
-		pci_dev_id = PCI_IDS_GAUDI2_SEC;
-
 	/* using vendor id as a magic number */
-	pci_dev_id |= (PCI_VENDOR_ID_HABANALABS << 16);
+	pci_dev_id = PCI_IDS_GAUDI2 | (PCI_VENDOR_ID_HABANALABS << 16);
 }
 
 u32 preboot_device_id_get(void)
